ros_trick/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
+
+TAG=latest
+ROS_TRICK_BRIDGE_IMAGE_NAME=ros_trick_bridge
+CANADARM_ROS_TRICK_DEMO_NAME=canadarm_ros_trick_demo
+export SPACE_ROS_IMAGE=osrf/space-ros:main
+CURRENT_PATH=$(pwd)
+SPACEROS_DOCKER_REPO_PATH="${CURRENT_PATH}/docker_repo"
+
+if [ ! -d ${SPACEROS_DOCKER_REPO_PATH} ]; then
+    mkdir -p ${SPACEROS_DOCKER_REPO_PATH}
+    git clone https://github.com/space-ros/docker.git ${SPACEROS_DOCKER_REPO_PATH}
+fi
```

Comment:
Can you add an `echo` that this is cloning this repo to that directory?

Also, should this just clone the latest on the default branch or will it be updated to a tag after the release?